std/manual.md

```diff
@@ -1638,7 +1635,7 @@ together.
 #### Utilize JSDoc
 
 All publicly exposed APIs and types, both via the `deno` module as well as the
-global/`window` namespace should have JSDoc documentation. This documentation is
+global `window` namespace should have JSDoc documentation. This documentation is
```

Comment:
This was fine I think.

Reply:
Ah, ok I might have misunderstood that one. I'll revert it.